workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/components/dashboard/utils/getChartColumns.ts

```diff
@@ -13,29 +13,34 @@ export default function getChartColumns(
 	const chartColumns: any[] = [];
 
 	const totalMDFActivitiesAmount = totalMDFRequested(
+		mdfCurrency,
```

Comment:
it might be clearer to name this something like useConvertedCurrency

Reply:
Could it be userCurrency? Because not always it will use the converted version.

Reply:
ya, either displayCurrency or userCurrency could work

Reply:
Done!